### PR TITLE
Added Multer to handle form-data

### PR DESCRIPTION
**Pull request by nhatnam23012000**
I added Multer to handle form-data and changed from express parser to body-parser

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
 const cors = require('cors')
 const apiRoute = require('./routers/router')
 
 dotenv.config()
 const app = express()
 const PORT = process.env.PORT || 3001
 
-app.use(express.json())
+app.use(bodyParser.urlencoded({ extended: true }))
+app.use(bodyParser.json())
```
Comment (honglax) [inline index.js]: If bodyParser is deprecated, please keep using 
```
app.use(express.urlencoded({ extended: true }))
app.use(express.json())
```

Comment (honglax) [reply index.js]: And please remove `bodyParser` from this project then update the yarn.lock file.

Comment (nhatnam23012000) [reply index.js]: I have solved above problems

**Review by honglax — APPROVED** [review]
LGTM 🚢

**Pull request merged**